apps/ledger-live-mobile/scripts/download-hermes-profile.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env zx
+
+import fs from "fs";
+
+// replace the package name com.ledger.live by com.ledger.live.debug in
+// AndroidManifest.xml as `react-native profile-hermes` is a bit too dumb to
+// get the right package name derived from the appIdSuffix
+const manifestPath = "./android/app/src/main/AndroidManifest.xml";
+const originalManifestContent = fs.readFileSync(manifestPath, "utf8");
+const newManifestContent = originalManifestContent.replace(
+  'package="com.ledger.live"',
+  'package="com.ledger.live.debug"',
+);
+
+fs.writeFileSync(manifestPath, newManifestContent);
+
+if (!fs.existsSync("./temp")) {
+  fs.mkdirSync("temp");
+}
+const destDirPath = fs.realpathSync("./temp");
```

Comment:
not sure realpath is needed since it is supposed to resolve the real path of symlinks

Reply:
It's needed because then `react-native profile-hermes` logs the path of the downloaded file, and it basically just appends the filename to the path we provide, so it logs a relative path which doesn't make sense if you're executing this from the root dir of `ledger-live`.
With this fix it logs the absolute path which is ok in any context